src/polycubed/src/server/Resources/Endpoint/ListResource.cpp

```diff
@@ -183,8 +184,8 @@ void ListResource::CreateReplaceUpdateWhole(
     }
     // check if the operation completed successfully and in case update the configuration
     if (isOperationSuccessful(resp.error_tag)) {
-      if (auto d = core_->get_cubes_dump()) {
-        d->UpdateCubesConfig(request.resource(), jbody, keys, op,
+      if (configuration::config.getCubesDumpEnabled()) {
```

Comment:
I would integrate this line with the previous (if openation is succesfully and configdumpenabled then...)